app/models/upload.rb

```diff
@@ -72,6 +72,18 @@ def self.last_uploads_rows
     uploads.sort_by { |upload| upload.csv_type.downcase }
   end
 
+  def self.latest_uploads
```

Comment:
Can we rename this?  I just noticed there is a `.last_uploads` too (used in this method), and so I naturally wonder what the difference is between `last uploads` and `latest uploads`.
How about we rename this something like.  `since_last_version_creation` (`Upload.since_last_version_creation`)?  Thoughts?

Reply:
renamed